README.md

```diff
@@ -196,6 +196,15 @@ Then administrator should make the following one time configurations using the a
 6. Add global property `orderentryowa.labOrderAutoExpireTimeInDays` with a value of `30` days or any other number
 7. Add global propery Lab Orderables Concept Set `orderentryowa.labOrderablesConceptSet`, whose value is the UUID of a concept set of Class LabSet and whose Set Members are other LabSet concept sets or concept of Class Test.
 
+### Additional features
+
+As of version 1.3.0, the module supports a collecting an "order reason" for Lab Orders.  Order reasons are specified at
```

Comment:
I'm planning on doing  release before merging this in, and this is my target next version, will update if that changes